ranges/fetchers/private.go

```diff
@@ -0,0 +1,22 @@
+package fetchers
+
+// PrivateFetcher implements the IPRangeFetcher interface for private network ranges.
+type PrivateFetcher struct{}
+
+func (f PrivateFetcher) Name() string {
+	return "Private"
+}
+func (f PrivateFetcher) Description() string {
+	return "Hardcoded IP ranges for private network ranges. Used in testing."
+}
+func (f PrivateFetcher) FetchIPRanges() ([]string, error) {
+
+	return []string{
+		"127.0.0.0/8",
```

Comment:
Would be nice if we could remove the overlapping IPS with localhost fetcher

Reply:
I wasn't sure about that, but I'll change it.